sdks/wasm/DebuggerTestSuite/DateTimeTests.cs

```diff
@@ -0,0 +1,61 @@
+using System;
+using System.Threading.Tasks;
+using Xunit;
+using System.Globalization;
+
+namespace DebuggerTests
+{
+	public class DateTimeList : DebuggerTestBase {
+
+		[Theory]
+		[InlineData ("en-US")]
+		[InlineData ("de-DE")]
+		[InlineData ("ka-GE")]
+		[InlineData ("hu-HU")]
+        
+        // Currently not passing tests. Issue #19743
+        // [InlineData ("ja-JP")]
+        // [InlineData ("es-ES")]
+		public async Task CheckDateTimeLocale (string locale) {
+			var insp = new Inspector ();
+			var scripts = SubscribeToScripts(insp);
+
+			await Ready();
+			await insp.Ready (async (cli, token) => {
+				ctx = new DebugTestContext (cli, insp, token, scripts);
+				var debugger_test_loc = "dotnet://debugger-test.dll/debugger-datetime-test.cs";
+
+				await SetBreakpoint (debugger_test_loc, 20, 3);
+				
+				var pause_location = await EvaluateAndCheck (
+					"window.setTimeout(function() { invoke_static_method ('[debugger-test] DebuggerTests.DateTimeTest:LocaleTest'," 
+					+ $"'{locale}'); }}, 1);",
+					debugger_test_loc, 20, 3, "LocaleTest",
+					locals_fn: (locals) => {
+                        DateTimeFormatInfo dtfi = CultureInfo.GetCultureInfo(locale).DateTimeFormat;
+						CultureInfo.CurrentCulture = new CultureInfo (locale, false);
+						DateTime dt = new DateTime (2020, 1, 2, 3, 4, 5);
+						string dt_str = dt.ToString();
+
+						var fdtp = dtfi.FullDateTimePattern;
+						var ldp = dtfi.LongDatePattern;
+						var ltp = dtfi.LongTimePattern;
+						var sdp = dtfi.ShortDatePattern;
+						var stp = dtfi.ShortTimePattern;
+
+						CheckString(locals, "fdtp", fdtp);
```

Comment:
These can be changed to:
```
await CheckProps (locals, new {
        fdtp   = TString (dtfi.FullDateTimePattern),
	ldp    = TString (dtfi.LongDatePattern),
	ltp    = TString (dtfi.LongTimePattern),
	sdp    = TString (dtfi.ShortDatePattern),
	stp    = TString (dtfi.ShortTimePattern),
	dt     = TValueType ("System.DateTime", dt.ToString ())
}, "locals", num_fields: 9);

await CheckDateTime (locals, "dt", dt);
```

This does the same tests as `CheckString` that you were doing.
Also, the `dt = TValueType` checks the description of the local variable `dt`. And the subsequent `CheckDateTime`, fetches the values of that `dt`, and compares that with the expected `dt`.